Latex/inhalt/Kapitel1.tex

```diff
@@ -12,7 +12,7 @@ \section{Einleitung}
 \section{Beispiele}
 %! "../vorlage/" ist normalerweise weder bei Pixel- noch bei Vektorgrafiken nötig. Die mitgelieferten Grafiken liegen in einem anderen Ordner
 \subsection{Pixelgrafiken}
-Siehe Quelltext.
+Siehe Quelltext.\fn{Dort wird das Kommando \striche{bild} aufgeführt. In dieser Vorlage sind allerdings keine Bilder (im vorgesehenen Ordner) enthalten.}
 %\bild[0.5]{../vorlage/ba-gc-logo}{Text zu einem Bild}{label1} %!bei Dateien in "bilder" ohne "../vorlage/"
```

Comment:
müsste an der stelle dann nicht der pfad zu bilder/ (also .) hin und nicht in den vorlagen-ordner? das ergibt denke wesentlich mehr sinn

Reply:
ich änder das im nächsten push